setup.cfg

```diff
@@ -34,6 +34,8 @@ project_urls =
 [options]
 zip_safe = False
 python_requires = >=3.7
+package_dir =
+    = src
```

Comment:
This looks strange.. Presumable it corresponds to `package_cir = {'': 'src'}` as explained [here](https://docs.python.org/3/distutils/setupscript.html#listing-whole-packages)

It seems to work though..